docs/reference/aggregations/bucket/composite-aggregation.asciidoc

```diff
@@ -348,7 +348,7 @@ GET /_search
 \... will sort the composite bucket in descending order when comparing values from the `date_histogram` source
 and in ascending order when comparing values from the `terms` source.
 
-====== Missing bucket
+*Missing bucket*
```

Comment:
This section should be at the same level than the `order` one so it should be `==== Missing bucket`